modular_skyrat/modules/vore/code/obj/belly.dm

```diff
@@ -0,0 +1,299 @@
+/obj/vbelly
```

Comment:
The belly object complicates code for no gain that i am able to see. I'd consider just turning this into a datum and putting the vored mob directly inside the predator mob.

Reply:
This is because without it I can't track if something (like an item that a prey might drop inside the belly) is going into the belly, or into the person.